### PR TITLE
Add guest table

### PR DESCRIPTION
**Pull request by Leana0**

### CODE REVIEW

File: .jpb/persistence-units.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nareknaltakyan1) [inline .jpb/persistence-units.xml:1]: please delete this file

File: src/main/resources/db/migration/V002__Initial_schema.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+create  table  guest (
+    guest_id bigserial primary  key Not null ,
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Initial_schema.sql:2]: for id use just "id" name, but not "guest_id"

File: src/main/resources/db/migration/V002__Initial_schema.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    first_name varchar(255) not null ,
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Initial_schema.sql:3]: please format document

File: src/main/resources/db/migration/V002__Initial_schema.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    last_name varchar(255),
+    passport varchar(255),
+    created  timestamp not null ,
+    deleted timestamp ,
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Initial_schema.sql:7]: please add UNIQUE, INDEXES constraints

File: src/main/resources/db/migration/V002__Initial_schema.sql [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    last_name  varchar(255),
+    passport   varchar(255),
+    created    timestamp             not null,
+    deleted    timestamp,
+    phone      varchar(255)          not null,
+    email      varchar(255)
+)
+
+
+ CREATE INDEX FK_1 ON guest
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Initial_schema.sql:14]: please use more suitable name for example index_guest_passport

File: src/main/resources/db/migration/V002__Initial_schema.sql [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+     (
+      passport
+         );
+
+
+
+SELECT
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Initial_schema.sql:21]: wtf, what is this?
I have asked you to add UNIQUE for passport and index for email, why you send me "select" statment

File: .jpb/persistence-units.xml (hunk 1 above)
Comment (nareknaltakyan1) [reply .jpb/persistence-units.xml:1]: I am asking you to delete this file second time

File: src/main/resources/db/migration/V002__Add_guest_table.sql [hunk 7]
```diff
@@ -0,0 +1,26 @@
+create table guest
+(
+    id         bigserial PRIMARY KEY NOT NULL,
+    first_name varchar(255)          NOT NULL,
+    last_name  varchar(255),
+    passport   varchar(255) UNIQUE   NOT NULL,
+    created    timestamp             NOT NULL,
+    deleted    timestamp,
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Add_guest_table.sql:8]: please add updated field

**Review by nareknaltakyan1 — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V002__Add_guest_table.sql [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    phone      varchar(255) UNIQUE   NOT NULL,
+    email      varchar(255) UNIQUE   NOT NULL
+)
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Add_guest_table.sql:11]: ;

File: src/main/resources/db/migration/V002__Add_guest_table.sql [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+
+CREATE INDEX idx_guest_passport ON guest
+    (
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Add_guest_table.sql:15]: please fix formatting

File: src/main/resources/db/migration/V002__Add_guest_table.sql [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+(
+    id         bigserial PRIMARY KEY NOT NULL,
+    first_name varchar(255)          NOT NULL,
+    last_name  varchar(255),
+    passport   varchar(255) UNIQUE   NOT NULL,
+    phone      varchar(255) UNIQUE   NOT NULL,
+    email      varchar(255) UNIQUE   NOT NULL,
+    created    timestamp             NOT NULL,
+    deleted    timestamp,
+    updated    timestamp
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Add_guest_table.sql:11]: NOT NULL

File: src/main/resources/db/migration/V002__Add_guest_table.sql [hunk 11]
```diff
@@ -0,0 +1,17 @@
+create table guest
+(
+    id         bigserial PRIMARY KEY NOT NULL,
+    first_name varchar(255)          NOT NULL,
+    last_name  varchar(255),
+    passport   varchar(255) UNIQUE   NOT NULL,
+    phone      varchar(255) UNIQUE   NOT NULL,
+    email      varchar(255) UNIQUE   NOT NULL,
```
Comment (nareknaltakyan1) [inline src/main/resources/db/migration/V002__Add_guest_table.sql:8]: remove NOT NULL

File: .jpb/persistence-units.xml (hunk 1 above)
Comment (Leana0) [reply .jpb/persistence-units.xml]: done

**Review by nareknaltakyan1 — APPROVED** [review]

**Pull request merged**